Example/FinancialConnections Example/FinancialConnections Example/Playground/PlaygroundViewModel.swift

```diff
@@ -0,0 +1,364 @@
+//
+//  PlaygroundViewModel.swift
+//  FinancialConnections Example
+//
+//  Created by Krisjanis Gaidis on 11/5/22.
+//  Copyright © 2022 Stripe, Inc. All rights reserved.
+//
+
+import Combine
+import Foundation
+import StripeFinancialConnections
+import SwiftUI
+import UIKit
+
+final class PlaygroundViewModel: ObservableObject {
+
+    let playgroundConfiguration = PlaygroundConfiguration.shared
+
+    var sdkType: Binding<PlaygroundConfiguration.SDKType> {
```

Comment:
there's a lot of repetitive code in this PR so it likely could be reduced, but that's de-scoped until there's extra time in the future